tests/ops/api/v1/endpoints/test_connection_config_endpoints.py

```diff
@@ -859,6 +913,17 @@ def test_delete_manual_webhook_connection_config(
             .first()
             is None
         )
+        assert (
+            mock_queue.called
+        ), "Deleting this last webhook caused 'requires_input' privacy requests to be queued"
```

Comment:
Just to confirm this is deleting the last webhook, so Fidesops is sure there's no more input required from the user before dispatch?

Reply:
Yes, the last webhook is deleted, so the privacy request just needs to be re-queued. Data is no longer required for the webhook. If any data was previously uploaded, it would be ignored.